puppet.py

```diff
@@ -27,6 +27,22 @@ def check_disabled():
         sudo('test -f {0} && echo DISABLED || echo ENABLED'.format(lockfile))
 
 @task
+def regenerate_certificate():
+    """Regenerate a node certificate"""
+    if len(env.hosts) > 1:
+      abort("This task should only be run on one host at a time.")
+    sudo('rm -f /etc/puppet/ssl/certs/{}.pem'.format(env.hosts[0]))
```

Comment:
I think using [`govuk_node_clean`](https://github.gds/gds/puppet/blob/44c2500f394c6397b7cdc6feb3b53bf01cd60b59/modules/govuk/files/bin/govuk_node_clean) would be a better option here, as it will take care of this for you.